app/gimlet/base.toml

```diff
@@ -123,7 +123,7 @@ features = ["gimlet"]
 name = "task-thermal"
 features = ["gimlet"]
 priority = 5
-max-sizes = {flash = 32768, ram = 8192 }
+max-sizes = {flash = 65536, ram = 8192 }
```

Comment:
what happened here?

Reply:
It got bigger and I wasn't excited about removing the limit (because I want to investigate) or figuring out a precise new one (because I'm changing the toolchain version a lot right now).